### PR TITLE
Added webhook based workflow

### PR DESCRIPTION
**Pull request by shreypandey**

### CODE REVIEW

File: api/app/handlers/v1/__init__.py [hunk 1]
```diff
@@ -25,9 +25,11 @@
 async def handle_webhook(webhook_data: str) -> AsyncGenerator[Flow, None]:
     plugin_uuid = extract_reference_id(webhook_data)
     if not plugin_uuid:
-        raise ValueError("Plugin UUID not found in webhook data")
+        return
```
Comment (sameersegal) [inline api/app/handlers/v1/__init__.py:28]: Add a comment saying that you are ignoring and why

**Review by sameersegal — APPROVED** [review]

**Comment by sameersegal** [comment]
Update backup.sql

**Comment by shreypandey** [comment]
@sameersegal Added DB dump

File: api/app/handlers/v1/__init__.py (hunk 1 above)
Comment (shreypandey) [reply api/app/handlers/v1/__init__.py:28]: Some webhooks send sample requests as a heartbeat. We need to ignore those requests. One way of doing that is ignore the requests with no `JB_IDENTIFIER`.

Comment (sameersegal) [reply api/app/handlers/v1/__init__.py:28]: No add this into the code for future reference.

**Review by sameersegal — APPROVED** [review]

**Pull request merged**